templates/home/layouts/home.html

```diff
@@ -32,7 +32,26 @@
 			{% include headers-includes/sitesearch.html %}
 		</div>
 	</div>
-{% include gcweb-menu/inc-sitemenu.html %}
+	<hr>
+	<div class="container">
+		<div class="row">
+			{%- unless page.nomenu -%}
+				{%- if page.auth -%}
+					<div class="col-sm-9">
+				{%- else -%}
+					<div class="col-sm-12">
+				{%- endif -%}
+				{% include gcweb-menu/inc-sitemenu.html %}
+			{%- endunless -%}
+		</div>
+		{%- if page.auth -%}
+			<div class="col-sm-3">
+				{% include authentication/authentication.html %}
+			</div>
+		{%- endif -%}
+		</div>
+	</div>
+<!-- {% include gcweb-menu/inc-sitemenu.html %} -->
```

Comment:
Can we leverage the jekyll include "header"?

Note: The comments that I made in header include is applicable here too.

Reply:
@duboisp Can we have a quick chat to clarify this comment?

Reply:
Disregard my comment. The pattern need to be duplicated because on the home page we don't like the top left FIP image.